proto/spawn_diff.proto

```diff
@@ -33,7 +33,6 @@ message Modified {
 message Diff {
   oneof diff {
     StringSetDiff tool_paths = 1;
-    FileSetDiff tool_contents = 2;
```

Comment:
Do we need to add `reserved 2` to the message?

Reply:
We are the only consumers of this proto so far, the `explain` command doesn't provide a user-facing way to produce it. We can think about adding a proto output flag at some point, but so far I would like to keep it experimental.